Test failure with PyPi package because of missing test.conf

**Describe the bug**
Tests fail when using the PyPi package

**To Reproduce**
Steps to reproduce the behavior:
1. Try to run tests with the PyPi package

**Expected behavior**
Tests to succeed

**Log files**
```
E       FileNotFoundError: [Errno 2] No such file or directory: '/home/bart/Documents/Git/alpine/aports/testing/py3-mycroft-messagebus-client/src/mycroft-messagebus-client-0.8.4/test/test.conf'
```

**Environment (please complete the following information):**
 - Device type: Desktop
 - OS: Alpine Linux
 - Mycroft-core version: 20.08

**Additional context**
The file is available in this Github repository, it just hasn't been included in the PyPi package.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,2 +1,3 @@
 include LICENSE.md
 include requirements.txt
+include test/test.conf

--- a/setup.py
+++ b/setup.py
@@ -36,7 +36,7 @@ def required(requirements_file):
 
 setup(
     name='mycroft-messagebus-client',
-    version='0.8.4',
+    version='0.8.5',
     packages=['mycroft_bus_client', 'mycroft_bus_client.client',
               'mycroft_bus_client.util'],
     package_data={

--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,7 @@ setup(
     url='https://github.com/MycroftAI/mycroft-messagebus-client',
     license='Apache-2.0',
     author='Mycroft AI, Åke Forslund',
-    author_email='devs@mycroft.ai, ake.forslund@mycroft.ai',
+    author_email='devs@mycroft.ai, ake.forslund@gmail.com',
     description='Mycroft Messagebus Client',
     long_description=long_desc,
     long_description_content_type="text/markdown",

